src/test/resources/features/ifs/ReferenceData.feature

```diff
@@ -70,11 +70,6 @@ Feature: Get Debt For all the SUPPORTED REGIMES
       | false           | 0                       | 0                    | 0       | 500000           | 500000             | 0                    | 500000             | <interestOnlyIndicator> |
     Examples:
```

Comment:
I don't know if it's deliberate, but in case it isn't, you're missing IFS rules from the spreadsheet.

For example:
```markdown
      | 2570      | 1090     | false                 |
```

Reply:
These are included in Scenario Outline: Interest Bearing Employer PAYE charges. Line 114 onwards